kube/pod_test.go

```diff
@@ -164,20 +164,28 @@ func TestPodGetEnvVars(t *testing.T) {
 	}
 
 	for _, sample := range samples {
-		defaults := map[string]string{"SOME_VAR": sample.input}
+		defaults := map[string]string{
+			"SOME_VAR": sample.input,
+			"ALL_VAR": "placeholder",
+		}
 
 		vars, err := getEnvVars(role, defaults)
 		assert.NoError(err)
 		assert.NotEmpty(vars)
 
-		found := false
+		founda := false
+		foundb := false
```

Comment:
Well, we could go with `foundSome` and `foundAll` ;) I was in quick mode for these.